climpred/utils.py

```diff
@@ -164,6 +181,28 @@ def convert_time_index(xobj, time_string, kind):
     return xobj
 
 
+def find_start_dates(init_pm, control, single_init):
```

Comment:
`init_pm` doesn't need to be an argument, since it isn't used in this function.

Reply:
Maybe a more descriptive function name too. I don't think `find_start_dates` communicates exactly what this is doing.

Reply:
Find_start_date_for_init?

Reply:
changed to `find_start_dates_for_given_init`